packages/pass-style/src/types.d.ts

```diff
@@ -228,4 +228,8 @@ export type CopyTagged<
  */
 export type InterfaceSpec = string;
 
+/**
+ * Consider this export deprecated.
+ * Import `Checker` directly from `'@endo/common/ident-checker.js'` instead.
+ */
```

Comment:
Is this comment still needed, or is the type itself now deprecate on its own?

Reply:
I don't think it hurts to have this comment.